src/api/app/policies/comment_policy.rb

```diff
@@ -0,0 +1,25 @@
+class CommentPolicy < ApplicationPolicy
+  def initialize(user, record)
```

Comment:
Yep, I already discussed this a few weeks ago with @hennevogel. I think the solution from @DavidKang is fine, otherwise you always need to check in the view ``` if User.current && policy(comment).destroy?```. We can discuss maybe in a future PR if we should handle not logged in users different in the ApplicationPolicy, otherwise it's getting really ugly when we want to use the view helper more often.
Long story, short: For me this is fine for now!